vgen/src/main/java/io/gapi/vgen/csharp/CSharpContextCommon.java

```diff
@@ -83,7 +83,15 @@ public String addImport(String namespace) {
   // the functionality, since snippets can't call static functions.
   public String underscoresToCamelCase(
       String input, boolean capNextLetter, boolean preservePeriod) {
-    return s_underscoresToCamelCase(input, capNextLetter, preservePeriod);
+    return s_underscoresToCamelCase(input, capNextLetter, preservePeriod, false);
+  }
+
+  public static String s_underscoresToCamelCase(String input) {
```

Comment:
I suspect this is due to the Java-Formatter tool that has been run over all code in this repo.